NEWS.md

```diff
@@ -30,7 +30,8 @@
 * `get_source_expressions()` no longer fails on R files that match a knitr pattern (#743, #879, #1406, @AshesITR).
 * Parse error lints now appear with the linter name `"error"` instead of `NA` (#1405, @AshesITR).  
   Also, linting no longer runs if the `source_expressions` contain invalid string data that would cause error messages
-  in other linters. 
+  in other linters.
+   + Also precludes hanging on Rmd files with some syntax errors (#1443, @MichaelChirico).
```

Comment:
IMO this should be a separate bullet.

Reply:
Thanks! I originally thought this was a regression introduced in dev. Confirmed the same issue is on 3.0.0, so split to its own item.